polyfill/lib/calendar.mjs

```diff
@@ -306,11 +308,10 @@ impl['iso8601'] = {
     if (fields.month !== undefined && fields.year === undefined && fields.monthCode === undefined) {
```

Comment:
I wonder if we have to remove `fields.year === undefined` from this condition in order to match the spec? I'm not sure.

Reply:
Okay, so for this, my understanding was that, it corresponds to [Step 10.a](https://tc39.es/proposal-temporal/#sup-temporal.calendar.prototype.datefromfields) -> [Bullet point 2 of CalendarResolveFields](https://tc39.es/proposal-temporal/#sec-temporal-calendarresolvefields), but even in the spec text without this change, it doesn't throw `TypeError` on `year === undefined` and it's unrelated to the change we made. So, I left it untouched, but I _don't think_ we need to check for year, as per the spec text.

Reply:
You're right, my mistake. The shape of the accepted property bags doesn't change, it's only whether `year` is ignored if `monthCode` is present.